_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+### Preparing for Adjustments
+* **Plan to cut content.** Online workshops often run more slowly than they do in-person; sometimes much more slowly. Decisions to cut back should be intentional and collaborative.
+    * Remember that skipping steps in a workflow can create problems with dependencies later on.
+* **Agree on your priorities.** Discussing key objectives and broader goals with your instructional team in advance will help everyone navigate difficult trade-offs when time becomes an issue during a workshop.
+* **Have a backup** for all roles and documents. Technical failures come in all shapes and sizes, and life can still happen, too!
+* **Review your pre-workshop surveys** using the links provided in your email from our workshop administrators. Knowing your audience will help you adjust appropriately to their needs!
+### Update your Software
+* Especially for Zoom, you will want to be using the most recent version and the same version as everyone else. Remind your learners to update as well!
+## 2. Workshop Mechanics
+* **Assign a "Zoom DJ"** to assume the "host" role in the meeting. A Zoom DJ will manage breakouts, waiting rooms, muting, and troubleshooting as needed. This role is typically played by a Supporting Instructor; if/when they choose to teach, plan to hand off this role to another Instructor.
+* **Customize profile names** to clearly indicate instructors and helpers. For example, a helper named Ada might change their profile to read "Helper: Ada (she/her)"
+* **Use Zoom features as you would use sticky notes**, and remember to call upon learners to use them regularly. 
+    * On Zoom, learners can choose "reactions" that include a green '✓' and a red 'x' icon. Unlike other reactions, these persist until cleared.
+* **Use breakout rooms** for socialization and activities whenever possible. These do take time and that may mean cutting content. The benefits to creating a supportive learning environment make this cost worthwhile.
+    * Using a shared timer can help, especially as learners get used to using it.
+* **Ask learners to keep the chat focused.** You may wish to provide them with an alternate venue to share social chat.
+* **Use a 3-step help response system**.
+    * Step 1: Learners message a helper, who helps via **direct messaging** in the conference chat if they can. 
+    * Step 2: Helpers communicate about questions they have received via **back-channel conversations** (even if resolved). This will elevate common questions that merit attention of the teaching Instructor and will allow others to advise if the problem has not been solved.
+    * Step 3: For difficult problems, invite the learner to **share their screen** for live support. Depending on the workshop size, frequency of the problem, etc., this may be done in the main room as a pause in the workshop or in a breakout room with a helper.
```

Comment:
The instructors can also use the remote control function in Zoom (if activated) to support learners